alacritty/src/cli.rs

```diff
@@ -253,7 +253,10 @@ impl Options {
         self.config.clone()
     }
 
-    pub fn into_config(self, mut config: Config) -> Config {
+    pub fn into_config(
+        self,
+        mut config: Config,
+    ) -> ::std::result::Result<Config, ConfigValidationError> {
```

Comment:
Do we really want to error here? We have the option to just print an error to stderr and then start in the default directory anyways. That seems like a better idea to me.